packages/cardano-services/src/Projection/migrations/1695899010515-pool-delisted-table.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { PoolDelistedEntity } from '@cardano-sdk/projection-typeorm';
+
+export class PoolDelistedTableMigration1695899010515 implements MigrationInterface {
+  static entity = PoolDelistedEntity;
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE TABLE "pool_delisted"
+                             (
+
+                                 "stake_pool_id" character(56)           NOT NULL,
+                                 CONSTRAINT "PK_pool_delisted_stake_pool_id" PRIMARY KEY ("stake_pool_id")
+                             )`);
+
+    await queryRunner.query(`ALTER TABLE "pool_delisted"
```

Comment:
Was this generated or did you add it manually? I noticed PoolDelistedEntity doesn't have `@OneToOne` so I thought it wouldn't generate the foreign key

Reply:
It was generated, but later on, I edited the entity file, and not generated again. I will look into it.

Reply:
I think we should keep the foreign key. Just saw this as a mismatch. We probably just need to re-add a decorator for the relation.

Reply:
Unfortunately, typeorm doesn't provide an elegant way to do what we want here  (see https://github.com/typeorm/typeorm/issues/4569). Since we don't benefit from having a foreign key, I am gonna remove it.

Reply:
Sure, it's fine since we're re-populating the entire table every time (no use of cascade etc.).

Out of curiosity, why didn't you want to use a typeorm relation?

Reply:
Having the relation is not needed and doesn't add any value. It only adds an extra layer/complexity in my opinion.